README.md

```diff
@@ -60,6 +60,16 @@ Install prerequisites for E2E tests
 
 Run `dotnet test test\AllTests.proj`
 
+## Build VS Tooling
+
+Prerequisites:
+- Visual Studio 2017 15.5 (or newer)
```

Comment:
I would imagine that it's possible to build with earlier VS builds, but anyone with old VS installs is going to have a bad time for other reasons, so I'd rather say it's not supported.